README.md

```diff
@@ -61,6 +61,7 @@ Options:
   -z|--gzip                            Enable gzip compression
   -b|--brotli                          Enable brotli compression (requires .NET Core 3+)
   -c|--cors                            Enable CORS (It will enable CORS for all origin and all methods)
+  --save                               Save specified options to .netconfig for subsequent runs.
```

Comment:
Could we make this `--save-options`?  I could forsee other things we might want to save in the future, like HTTPS certs.

Reply:
Good call 👍